tests/backup/backup_resiliency_test.go

```diff
@@ -1418,6 +1421,7 @@ var _ = Describe("{ScaleDownPxBackupPodWhileBackupAndRestoreIsInProgress}", func
 			for _, namespace := range appNamespaces {
 				for i := 0; i < numberOfBackups; i++ {
 					sem <- struct{}{}
+					time.Sleep(sleepToGetUniqueTimeStamp)
 					backupName := fmt.Sprintf("%s-%s-%d-%v", BackupNamePrefix, namespace, i, time.Now().Unix())
```

Comment:
How about time.Now().Unix() + 2?

Reply:
The main intention of sleep is not to get unique time stamp.
The error I was facing is that when we create backups is parallel , I was seeing error:
 **[Failed to get diff extents sts:-22]]** 
So , as suggested by Mithun, we have to sleep between 2 backups

since I am already using index 'i' in backup name..it will be unique anyway
have removed the word causing confusion and directly giving sleel